### ISSUE TITLE
Native module creates a large glob pattern when building universal macOS installer

### ISSUE THREAD

**Issue by alexwkleung** [alexwkleung/Iris#87]
When building the universal installer for macOS, the native module in dependencies create a large glob pattern that electron-builder cannot handle. Building installers separately for x64 and arm64 work fine.

```bash
  • electron-builder  version=23.6.0 os=22.6.0
  • loaded configuration  file=package.json ("build" field)
  • writing effective config  file=dist/builder-effective-config.yaml
  • packaging       platform=darwin arch=x64 electron=28.0.0 appOutDir=dist/mac-universal--x64
  • packaging       platform=darwin arch=arm64 electron=28.0.0 appOutDir=dist/mac-universal--arm64
  • packaging       platform=darwin arch=universal electron=28.0.0 appOutDir=dist/mac-universal
  ⨯ pattern is too long  failedTask=build stackTrace=TypeError: pattern is too long
    at assertValidPattern (/Users/alex/Desktop/Git/Iris/node_modules/minimatch/minimatch.js:279:11)
    at minimatch (/Users/alex/Desktop/Git/Iris/node_modules/minimatch/minimatch.js:116:3)
    at handleFile (/Users/alex/Desktop/Git/Iris/node_modules/asar/lib/asar.js:123:26)
    at next (/Users/alex/Desktop/Git/Iris/node_modules/asar/lib/asar.js:148:11)
    at next (/Users/alex/Desktop/Git/Iris/node_modules/asar/lib/asar.js:149:12)
    at Object.exports.mergeASARs (/Users/alex/Desktop/Git/Iris/node_modules/@electron/universal/src/asar-utils.ts:198:5)
    at exports.makeUniversalApp (/Users/alex/Desktop/Git/Iris/node_modules/@electron/universal/src/index.ts:205:7)
    at MacPackager.doPack (/Users/alex/Desktop/Git/Iris/node_modules/app-builder-lib/src/macPackager.ts:125:9)
    at MacPackager.pack (/Users/alex/Desktop/Git/Iris/node_modules/app-builder-lib/src/macPackager.ts:192:7)
    at Packager.doBuild (/Users/alex/Desktop/Git/Iris/node_modules/app-builder-lib/src/packager.ts:441:9)
    at Object.executeFinally (/Users/alex/Desktop/Git/Iris/node_modules/builder-util/src/promise.ts:12:14)
    at Packager._build (/Users/alex/Desktop/Git/Iris/node_modules/app-builder-lib/src/packager.ts:376:31)
    at Packager.build (/Users/alex/Desktop/Git/Iris/node_modules/app-builder-lib/src/packager.ts:337:12)
    at Object.executeFinally (/Users/alex/Desktop/Git/Iris/node_modules/builder-util/src/promise.ts:12:14)
make: *** [build-mac-universal] Error 1
```

**Comment by alexwkleung** [comment]
Reference in #63

**Comment by alexwkleung** [comment]
Fixed as of 0733000.

**Comment by alexwkleung** [comment]
The issue was that the build entry in `package.json` took precedence over `electron-builder.yml`. Therefore it only loaded the glob patterns in `package.json` while ignoring all the patterns set in `electron-builder.yml`.
